code/_globalvars/bitfields.dm

```diff
@@ -189,7 +189,7 @@ GLOBAL_LIST_INIT(bitfields, list(
 		"HAND_LEFT" = HAND_LEFT,
 		"HAND_RIGHT" = HAND_RIGHT
 		),
-	"flags_armor_protection" = list(
+	"body_parts_covered" = list(
```

Comment:
if this is just a variable name change then atomise this out of the PR so its not fucking 81 files